config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```
@@ -91,7 +91,7 @@ periodics:
         --gcp-zone=us-west1-b
         --node-tests=true
         "--node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/image-config.yaml --extra-envs=${EXTRA_ENVS}"
-        '--node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"'
+        '--node-test-args=--kubelet-flags="--container-runtime=containerd --container-runtime-pid-file= --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock --cgroups-per-qos=true --cgroup-root=/"'
```

```suggestion
        '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
```

would match our containerd jobs.

The only valid option for container-runtime is "remote", "containerd" is not possible so this will fail if you specify that.